pinot-spi/src/main/java/org/apache/pinot/spi/config/table/QueryConfig.java

```diff
@@ -37,15 +37,23 @@
   // If the server times out, it will directly interrupt the query execution. The server response does not matter much
   // because by the time the server times out, the broker should already timed out and returned the response.
   private final Long _timeoutMs;
+  private final Boolean _serveOfflineSegmentsImmediately;
```

Comment:
Can you beef up the documentation to this config in the java class. Also, are the pinot docs in repo as well? Should be updated to reflect the new type of config you can set.

Reply:
Not finding where the docs reside within repo, but I've added documentation to the config class.